src/components/Dashboard/Category-Card/CardModal.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
```

Comment:
This file is getting a little on the longer side for a React component.  If we're getting up to 100 lines of code, it's USUALLY a sign that we could simplify via refactoring or encapsulating or both. Consider if there are functions or variables that could be encapsulated in their own file for import (getValidFlightData, getCurrentAnalytics, setStatusColor all seem to be good candidates for this).  What would you have to refactor, and what would that file have to have closure over for the exported methods to be correct?